### ISSUE TITLE
Start a server from a configuration

### ISSUE THREAD

**Issue by barischrooneyj** [barischrooneyj/simple-store#5]
Create a function that can serve the database with an endpoint for each of the minimum `Store` methods

- [x] viewAll TypeKey
- [x] setAll TypeKey
- [x] rmAll TypeKey

**Comment by barischrooneyj** [comment]
Something like:

``` haskell

type Server API = ...

serveStore :: Store s => s -> Server StoreAPI
serveStore ...
```

**Comment by barischrooneyj** [comment]
Closed by #085b7b14efb5f2ce3c5238e78e2e067a74ccbc17
